docs/apidocs/qiskit_addon_obp.utils.operations.rst

```diff
@@ -1,10 +1,14 @@
-==========
-operations
-==========
-
-.. _qiskit_addon_obp-utils-operations:
+=============================================================
+Operator utilities (:mod:`qiskit_addon_obp.utils.operations`)
+=============================================================
 
 .. automodule:: qiskit_addon_obp.utils.operations
    :no-members:
    :no-inherited-members:
    :no-special-members:
+
+.. currentmodule:: qiskit_addon_obp.utils.operations
+
+.. autofunction:: apply_op_to
+.. autofunction:: to_global_op
+.. autofunction:: reduce_op
```

Comment:
We appear to be missing the following:
```suggestion
.. autofunction:: reduce_op
.. autofunction:: apply_reset_to
```

Reply:
It looks like the current docs do not have `apply_reset_to`. I'll add it here :) Although note that is adding a new API.

Reply:
I think this was just an oversight on our end as it should already be documented. I am fine with adding that as part of this PR without acknowledging this change explicitly.

@caleb-johnson any objections?

Reply:
Ahh yes, agree with @mrossinek . This should've already been in place. Thanks!